lib/hmmlearn/base.py

```diff
@@ -253,6 +253,64 @@ def score(self, X, lengths=None):
         """
         return self._score(X, lengths, compute_posteriors=False)[0]
 
+    def score_batches(self, X, lengths=None):
```

Comment:
Are there similar APIs in scikit-learn or related packages?  If yes, how are they named?

Reply:
How about naming this `score_samples`, it is what the GaussianMixtureModel in sklearn provides.

Reply:
The problem is that a "sample" would typically be a single observation (and the confusion typically doesn't occur in sklearn); perhaps score_sequences?  (i.e. implicitly "score the individual sequences")

Reply:
That, and there already exists a `score_samples` method ;-) 

+1 for `score_sequences`.

Reply:
`score_sequences` sounds good to me too. There isn't really a similar API in sklearn I think.